### PR TITLE
RequestInfo uses new cognito

### PR DESCRIPTION
**Pull request by axthosarouris**

### CODE REVIEW

File: apigateway/src/main/java/nva/commons/apigateway/ApiGatewayHandler.java [hunk 1]
```diff
@@ -8,6 +8,7 @@
 import com.fasterxml.jackson.core.JsonProcessingException;
 import com.fasterxml.jackson.databind.ObjectMapper;
 import com.google.common.net.HttpHeaders;
+import com.google.common.net.MediaType;
```
Comment (axthosarouris) [inline apigateway/src/main/java/nva/commons/apigateway/ApiGatewayHandler.java:11]: This is only reformatting

File: apigateway/src/test/java/nva/commons/apigateway/RequestInfoTest.java [hunk 2]
```diff
@@ -47,25 +56,13 @@
     public static final String PATH_DELIMITER = "/";
     public static final int UNNECESSARY_ROOT_NODE = 0;
     public static final int FIRST_NODE = 0;
-    public static final String accessRightSample2 = "REJECT_DOI_REQUEST";
```
Comment (axthosarouris) [inline apigateway/src/test/java/nva/commons/apigateway/RequestInfoTest.java:50]: obsolete constants due to new cognito setup

File: auth/src/main/java/no/unit/nva/auth/FetchUserInfo.java [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+import java.net.URI;
+import java.net.http.HttpClient;
+import java.net.http.HttpRequest;
+import java.net.http.HttpResponse;
+import java.net.http.HttpResponse.BodyHandlers;
+import nva.commons.core.paths.UriWrapper;
+
+public class FetchUserInfo {
+
+    public static final String OAUTH_USER_INFO_ENDPOINT = "oauth2/userInfo";
```
Comment (axthosarouris) [inline auth/src/main/java/no/unit/nva/auth/FetchUserInfo.java:13]: Trying to keep the dependencies light and not include a library for the Authorization header

File: auth/src/main/java/no/unit/nva/auth/JsonConfig.java [hunk 4]
```diff
@@ -0,0 +1,24 @@
+package no.unit.nva.auth;
+
+import static nva.commons.core.attempt.Try.attempt;
+import com.fasterxml.jackson.jr.annotationsupport.JacksonAnnotationExtension;
+import com.fasterxml.jackson.jr.ob.JSON;
+import nva.commons.core.JacocoGenerated;
+
+@JacocoGenerated
+public final class JsonConfig {
```
Comment (axthosarouris) [inline auth/src/main/java/no/unit/nva/auth/JsonConfig.java:9]: Class for making it easier to move to another json library if necessary

File: nvatestutils/src/main/java/no/unit/nva/stubs/WiremockHttpClient.java [hunk 5]
```diff
@@ -0,0 +1,97 @@
+package no.unit.nva.stubs;
+
```
Comment (axthosarouris) [inline nvatestutils/src/main/java/no/unit/nva/stubs/WiremockHttpClient.java:2]: A copy paste of the WiremockHttpClient that we have implemented in many other modules

**Review by brinxmat — DISMISSED** [review]

**Pull request merged**